src/Formatter/Output.php

```diff
@@ -5,21 +5,11 @@
 class Output
 {
     public const NO_FILE = '/dev/null';
-
-    /** @var bool */
-    private $noOutput = false;
-
-    /** @var string */
-    private $stdOutFile;
-
-    /** @var bool */
-    private $stdOutAppend;
-
-    /** @var string */
-    private $stdErrFile;
-
-    /** @var bool */
-    private $stdErrAppend;
+    private bool $noOutput = false;
+    private string $stdOutFile;
```

Comment:
This looks like it could be null
```suggestion
    private ?string $stdOutFile = null;
```